miio/gateway/devices/subdevice.py

```diff
@@ -220,6 +224,9 @@ def get_battery(self):
     @command()
     def get_voltage(self):
         """Update the battery voltage, if available."""
+        if not self._battery_powered:
+            return None
```

Comment:
I'm fine with this, but maybe add a debug log just in case someone starts to wonder why they are just getting None as a response. Also, adding type hints (here `Optional[int]`?) would be helpful.

Reply:
done, thanks for the tips